.travis.yml

```diff
@@ -28,10 +28,9 @@ matrix:
       language: generic
 
 before_install:
+ - pyenv versions
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source .travis-osx; fi
- # work around https://github.com/travis-ci/travis-ci/issues/8363
- - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then pyenv global system 3.5; fi
-
+ 
```

Comment:
This line breaks the CI lint check - fixed in https://github.com/tox-dev/tox/pull/718.

Reply:
thanks!